qa/workunits/fs/quota/quota.sh

```diff
@@ -97,12 +97,14 @@ stat --printf="%n %s\n" ancestor/p1/p2/parent/p3/file1
 #get/set attribute
 
 setfattr -n ceph.quota.max_bytes -v 0 .
-setfattr -n ceph.quota.max_bytes -v 1 .
-setfattr -n ceph.quota.max_bytes -v 9223372036854775807 .
-expect_false setfattr -n ceph.quota.max_bytes -v 9223372036854775808 .
+expect_false setfattr -n ceph.quota.max_bytes -v 1 .
+setfattr -n ceph.quota.max_bytes -v 4096 .
+expect_false setfattr -n ceph.quota.max_bytes -v 0x7FFFFFFFFFFFFFFF .
```

Comment:
This can now be specified with human readable (K, M, G, etc..)

Reply:
Yeah, make sense. Will revise it.